docs/src/sequencepaxos/storage.md

```diff
@@ -91,7 +91,7 @@ An in-memory storage implementation, `MemoryStorage` offers fast reads and write
 ```
 
 ## PersistentStorage
```

Comment:
Could you add a subsection here that shows how to do fail-recovery? I.e. you re-intiate the persistent storage and recreate a SequencePaxos instance and then call `seq_paxos.fail_recovery()`